### ISSUE TITLE
Internal server error in Playout after pulling git

### ISSUE THREAD

**Issue by matias-tecnosoul** [LibreTime/libretime#1565]
Hi, yesterday I pulled git and started to get this Internal server errors in libretime-playout:

```
Feb  4 10:00:29 guayraradio libretime-playout[334]: Payload: None
Feb  4 10:00:29 guayraradio libretime-playout[334]: Response:
Feb  4 10:00:29 guayraradio libretime-playout[334]: 2022-02-04 10:00:29,909 [main] [ERROR]  500 Server Error: Internal Server Error for url: http://localhost:80/api/register-component/format/json/api_key/CK00RPNCVHPDBW9DESPQ/component/pypo
Feb  4 10:00:40 guayraradio libretime-playout[334]: 2022-02-04 10:00:40,000 [utils] [ERROR]  GET http://localhost:80/api/register-component/format/json/api_key/CK00RPNCVHPDBW9DESPQ/component/pypo request failed '500':

```

I tried with wget:

```
 wget http://localhost:80/api/register-component/format/json/api_key/CK00RPNCVHPDBW9DESPQ/component/pypo
--2022-02-04 10:16:34--  http://localhost/api/register-component/format/json/api_key/CK00RPNCVHPDBW9DESPQ/component/pypo
Resolving localhost (localhost)... ::1, 127.0.0.1
Connecting to localhost (localhost)|::1|:80... connected.
HTTP request sent, awaiting response... 500 Internal Server Error
2022-02-04 10:16:34 ERROR 500: Internal Server Error.
```
I can see the "On Air" going red for a few seconds after each track change.

**LibreTime version:**
9fc4552cf

- OS: Ubuntu Bionic Beaver

Any hints please? thanks!

**Comment by matias-tecnosoul** [comment]
I've just seen the pinned issue "master branch to main ".
Just in case it helps, I first pulled git from inside the libretime folder that was already installed and got some errors about the branch. 
I opted to download a fresh copy and installed with ./install -fiap -d

**Comment by jooola** [comment]
The main branch received a lot of breaking changes in the past weeks. Check for commits with a BREAKING CHANGE footer to get instructions.

**Comment by matias-tecnosoul** [comment]
thanks for the tip. I checked them and created some new dirs and files missing (in logs...) but nothing.
I pulled git again and got some updates. Then I started getting "Failed to insert Cors URL; database isn't configured properly!
". However the part of the db conf in the installer passed ok.
I see this in postgres log:
```
2022-02-04 15:12:30.419 UTC [11708] airtime@airtime FATAL:  password authentication failed for user "airtime"
2022-02-04 15:12:30.419 UTC [11708] airtime@airtime DETAIL:  Password does not match for user "airtime".
	Connection matched pg_hba.conf line 94: "host    all             all             ::1/128                 md5"

```
Looked into /etc/postgresql/10/main/pg_hba.conf but cant really find anything yet. Will check that file's meaning.
I also tried uninstalling but same error.
thanks

**Comment by jooola** [comment]
I guess you will have to give us some logs, check the apache error log file.

**Comment by matias-tecnosoul** [comment]
apache error log:

`[php7:notice] [pid 770] [client 10.0.3.161:51684] PHP Notice:  Undefined index: dbname in /usr/share/airtime/php/legacy/build/airtime-setup/forms/database-settings.php on line 31, referer: https://guayraradio.tecnosoul.com.ar/embed/weekly-program
`
I'm looking for other logs...

**Comment by matias-tecnosoul** [comment]
Regarding the pg_hba.conf error, I'm comparing it with an older LT install running in another machine.
Old install:
```

# Database administrative login by Unix domain socket
#local   all             postgres                                peer
local   all             postgres                                md5
# TYPE  DATABASE        USER            ADDRESS                 METHOD

# "local" is for Unix domain socket connections only
local   all             all                                     md5
# IPv4 local connections:
host    all             all             127.0.0.1/32            md5
# IPv6 local connections:
host    all             all             ::1/128                 md5
# Allow replication connections from localhost, by a user with the
# replication privilege.
local   replication     all                                     peer
host    replication     all             127.0.0.1/32            md5
host    replication     all             ::1/128                 md5

```
and current one:
```
local   all             postgres                                trust

# TYPE  DATABASE        USER            ADDRESS                 METHOD
 "local" is for Unix domain socket connections only
local   all             all                                     peer
# IPv4 local connections:
host    all             all             127.0.0.1/32            md5
# IPv6 local connections:
host    all             all             ::1/128                 md5
# Allow replication connections from localhost, by a user with the
# replication privilege.
local   replication     all                                     peer
host    replication     all             127.0.0.1/32            md5
host    replication     all             ::1/128                 md5

```

**Comment by matias-tecnosoul** [comment]
Well, I've decided to move this radio to a fresh OS. 
I've installed a Debian Bullseye from scratch, restored the db and files and it's woking now!! :)
So it seemed my Focal install was broken somewhere. Oh well, it was time to upgrade the server...

I did have to adjust postgres /etc/postgresql/13/main/pg_hba.conf again:
from:
`host    all             all             ::1/128                 md5
`
to:
`host    all             all             ::1/128                 trust
`
I'm not sure if it's a security issue. 
Thanks a lot for your help!
